scout/server/blueprints/clinvar/views.py

```diff
@@ -116,3 +116,12 @@ def clinvar_download_csv(submission, csv_type, clinvar_id):
         mimetype="text/csv",
         as_attachment=True,
     )
+
+
+@clinvar_bp.route("/<submission>/download/json/<clinvar_id>", methods=["GET"])
+def clinvar_download_json(submission, clinvar_id):
+    """Download a json for a clinVar submission"""
+
+    code, conversion_res = controllers.json_api_submission(submission_id=submission)
```

Comment:
This function does the conversion and returns the json to the javascript code that has been calling it I guess. Try to make it return directly the file as a response. 

Other thing, the downloaded file doesn't have indentation if you open it now:

<img width="1216" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/048b6163-8a63-4527-96e9-45d625a38758">

Would be more helpful if you could pre-format the json prior to return it in the response. Check here how to do it: https://stackoverflow.com/questions/9170288/pretty-print-json-data-to-a-file-using-python